stdlib/2and3/argparse.pyi

```diff
@@ -117,6 +117,7 @@ class Action:
                  option_string: _Text = ...) -> None: ...
 
 class Namespace:
+    def __init__(self, **kwargs: Any) -> None: ...
     def __getattr__(self, name: _Text) -> Any: ...
     def __setattr__(self, name: _Text, value: Any) -> None: ...
```

Comment:
Looking at the code I noticed it also has `def __contains__(self, key: str) -> bool: ...`; would you mind adding that?

Reply:
done

Reply:
Thanks! I'll merge once Travis passes (unless I forget).